Source/Particles/Gather/GatherExternalPulsarFieldOnGrid.H

```diff
@@ -0,0 +1,562 @@
+/* Copyright 2019 Axel Huebl, David Grote, Maxence Thevenet
```

Comment:
Okay, so admittedly I didn't actually check all the indexing for the shape factors.

Reply:
These were similar to the shape-factor implementation for particles from the self-consistent fields. :)